src/store/panesSettings/chart.ts

```diff
@@ -45,7 +47,9 @@ const state = {
   showHorizontalGridlines: false,
   horizontalGridlinesColor: 'rgba(255,255,255,.1)',
   showVerticalGridlines: false,
-  verticalGridlinesColor: 'rgba(255,255,255,.1)'
+  verticalGridlinesColor: 'rgba(255,255,255,.1)',
+  showWatermark: true,
+  watermarkColor: 'rgba(255,255, 0.1)'
```

Comment:
Little typo here `rgba(255,255, 0.1)` should be `rgba(255,255,255, 0.1)`
Then good to go I think 👍🏼

Reply:
THATS WHY IT WAS YELLOW!! Thank you!